lists/1313161554/list.json

```diff
@@ -865,6 +865,14 @@
       "decimals": 24,
       "chainId": 1313161554,
       "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0x06aEBB0f3D9eBe9829E1B67bD3dd608F711D3412/logo.svg"
+    },
+    {
+      "name": "NearStarter Index Pool",
+      "address": "0x11fffdcb623b4882daed70eb22c2bf51215178dc",
```

Comment:
Addresses need to be checksummed please
 be sure to fix it in the .json and the logo directory